itez/beneficiary/uuid.py

```diff
@@ -0,0 +1,10 @@
+import uuid
+
+
+def generate_uuid_and_agent_code():
+    """
+     This function generate a 7 digit code for agent.
+    """
+    uuid_one = uuid.uuid4()
+    agent_code = str(uuid_one)[1:8].upper()
```

Comment:
for this lets get the mids values between the `-something found here-` not the first 7

Reply:
moved uuid.py to utils.py